src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -85,8 +84,7 @@ private[redshift] case class RedshiftRelation(
   }
 
   override def buildScan(requiredColumns: Array[String], filters: Array[Filter]): RDD[Row] = {
-    val creds =
```

Comment:
This is the callsite which was missing the `params.temporaryAWSCredentials.getOrElse()` call.